### PR TITLE
r?fzzzy More misc fixes

### PR DESCRIPTION
**Pull request by ianb**

### CODE REVIEW

File: addon/data/shooter-interactive-worker.js [hunk 1]
```diff
@@ -477,10 +463,26 @@ function addHandlers() {
...
+    self.port.emit("deactivate");
+  }
+  if (event.key === "Enter") {
+    self.port.emit("take-shot");
+  }
 }
 
 function removeHandlers() {
-  for (let name of registeredDocumentHandlers) {
+  for (let name in registeredDocumentHandlers) {
```
Comment (fzzzy) [inline addon/data/shooter-interactive-worker.js]: Why this change? I think "of" is correct. With "in", name will be 0, 1, 2, etc

Comment (fzzzy) [inline addon/data/shooter-interactive-worker.js]: Never mind, i see that it is not an array

Comment (ianb) [inline addon/data/shooter-interactive-worker.js]: Of course at one time it *was* an array, hence the bug...
